module/x/gravity/cosmos_originated_test.go

```diff
@@ -38,12 +38,14 @@ type testingVars struct {
 	t                  *testing.T
 }
 
+const AddrLen = 20
```

Comment:
we can use this const from either the SDK or geth types

Reply:
this const isn't in the SDK anymore.